docs/backend/internationalization.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Internationalization (i18n)
+---
+
+# Internationalization (i18n)
+
+## What is internationlization (i18n)?
+
+To over simplify the concept a bit, internalization (i18n for short) is the
+process of making the platform more user-friendly in various languages for
+people around the globe. This includes, but is certainly not limited to, things
+like making the site available in different languages, changing currency values
+to match your region, changing date formats, etc.
+
+## What do we currently support?
+
+Currently, we allow users to set some language settings (beta) under
+`/settings/misc`. Based on your selection, the feed algorithm will prioritize
+content written in your preferred languages.
+
+We also introduced some routing to lay the groundwork for a more comprehensive
+i18n implementation.
+
+## What is the goal?
+
+We want everyone to feel included, regardless of where they're located or what
+language(s) they speak. The goal is to make the platform available in various
+languages.
+
+## How do you get involved?
+
+The following is a high level outline of an approach to internationalization.
+This is by no means set in stone.
+
+We encourage you to open a pull request (PR) to this documentation or to
+contribute to internationalization with your ideas - we're
+[open-source](https://github.com/forem/forem/pulls)!
+
+## Routing
+
+We have logic for routes setup. You can visit a page and add `/locale/:locale`
+to the beginning of the path. For example, if you visit the homepage, you can
+add `/locale/fr-ca` for French, Canadian where `fr` is the language code and
```

Comment:
```suggestion
add `/locale/fr-ca/` for French, Canadian where `fr` is the language code and
```
Suggesting this only because I wonder if devs will just copypasta this to the end and not actually read and see that it's meant to be prepended to the _beginning_, as you describe in the docs. Not a blocker though, just an idea :)